aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -193,9 +193,7 @@ module aptos_framework::account {
     //  1. First, append the Ed25519 scheme identifier '0x00' to `account_public_key_bytes`
```

Comment:
could we update the doc as well?

Reply:
What doc?

Reply:
(The authentication key derivation is already in our public facing docs: https://aptos.dev/concepts/basics-accounts#single-signer-authentication)